terraform/modules/aws-dns-records/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "fqdns" {
+  value = [for v in var.subdomains: join(".", concat([v], local.name_suffix))]
```

Comment:
This replicates the same logic used in https://github.com/wireapp/wire-server-deploy/blob/develop/terraform/modules/aws-dns-records/resources.route53.tf#L5 and https://github.com/wireapp/wire-server-deploy/blob/develop/terraform/modules/aws-dns-records/resources.route53.tf#L16.

Wouldn't it make more sense to move logic from these places into a variable, and export these variable values (and/or a combined list of it) as output?

Reply:
Since the function call in those two lines refer to diferent context on each iteration, moving it into a `local` (if that's what you mean by *variable*) approach wouldn't really work iMHO. But concatenating the name from each resource might work. Let's see if TF handles `for_each = []` as expected ...

Reply:
What about now?

Reply:
Neat!